composer.json

```diff
@@ -106,6 +106,7 @@
         "symfony/finder": "6.4.24",
         "symfony/http-client": "6.4.26",
         "symfony/http-foundation": "6.4.26",
+        "symfony/http-kernel": "^7.2",
```

Comment:
hi @kojiromike , this version does not look right. pretty sure symfony stuff should be 6.4.* for OpenEMR (since using long term release of symfony)

Reply:
It was already `7.2.9` at `composer.lock` https://github.com/openemr/openemr/pull/9279/files#diff-f37acfaa6b11f575a9a6f41a75fa73a61d0f8ebc2c9b8cddc215d8aca10e44f5L14313-L14314